documentation/modules/ROOT/pages/connectors/mongodb.adoc

```diff
@@ -389,24 +456,29 @@ When we compare this to the value in the _insert_ event, we see a couple of diff
 * The `op` field value is now `u`, signifying that this document changed because of an update
 * The `patch` field appears and has the stringified JSON representation of the actual MongoDB idempotent change to the document, which in this example involves setting the `first_name` field to a new value
 * The `after` field no longer appears
-* The `source` field structure has the same fields as before, but the values are different since this event is from a different position in the oplog.
-* The `ts_ms` shows the timestamp that Debezium processed this event.
+* The `source` field structure has the same fields as before, but the values are different since this event is from a different position in the oplog
+* The `ts_ms` shows the timestamp that {ProductName} processed this event
 
 [WARNING]
 ====
 The content of the `patch` field is provided by MongoDB itself and its exact format depends on the version.
+ifndef::cdc-product[]
 You can thus expect that the messages will not be same for MongoDB 3.4 and 3.6 and you should be careful
 while upgrading the MongoDB instance to a new version.
 
 All examples in this document were obtained from MongoDB 3.4 and might differ if you use a different one.
```

Comment:
@gunnarmorling Should we consider updating the examples to use whatever downstream MongoDB versions are supported for ease of maintainability here?  I'm not aware of any specific differences but given we call this out there may obviously be some?

Reply:
In fact I think this one ("You can thus expect that the messages will not be same for MongoDB 3.4 and 3.6") is actually a bit too specific.  Rather let's say something like:

> The content of the `patch` field is provided by MongoDB itself and its exact format depends on the specific database version. You should therefore be prepared for potential changes to the format when upgrading the MongoDB instance to a new version.

@Naros, are you saying we should have different example versions? I think having one is good, naming its specific version (as we do) and based on the general note above.